user.go

```diff
@@ -172,3 +241,92 @@ func importUserV1(source map[string]interface{}) (*user, error) {
 	return result, nil
 
 }
+
+func importUserV2(source map[string]interface{}) (*user, error) {
+	fields := schema.Fields{
```

Comment:
Yes, good point. I don't like the idea of assuming the fields are always growing version after version but I guess this may be more elegant this time.